From 3c6f1496fa3eae718305b5ddadfe85a7641ed51a Mon Sep 17 00:00:00 2001
From: Geoffrey Papaux <geoffrey.papaux@gmail.com>
Date: Wed, 20 Nov 2013 16:53:12 +0100
Subject: [PATCH 2/2] omap5-kvm: patch for lpae support in platform drivers

This is needed for USB/Network support on omap5-uevm board.
Patch from https://bugs.launchpad.net/arndale/+bug/1088845
---
 drivers/of/platform.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/platform.c b/drivers/of/platform.c
index f6dcde2..814f8c2 100644
--- a/drivers/of/platform.c
+++ b/drivers/of/platform.c
@@ -214,7 +214,7 @@ static struct platform_device *of_platform_device_create_pdata(
 #if defined(CONFIG_MICROBLAZE)
 	dev->archdata.dma_mask = 0xffffffffUL;
 #endif
-	dev->dev.coherent_dma_mask = DMA_BIT_MASK(32);
+	dev->dev.coherent_dma_mask = DMA_BIT_MASK(64);
 	dev->dev.bus = &platform_bus_type;
 	dev->dev.platform_data = platform_data;
 
-- 
1.7.9.5

